docs/ruleset_functions/is_baseline_system_1(HVAC_id).md

```diff
@@ -0,0 +1,48 @@
+# is_baseline_system_1  
+
+**Description:** Get either Sys-1, Sys-1a, Sys-1b, Sys-1c, or Not_Sys_1 string output which indicates whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 1 (PTAC), system 1a (system 1 with purchased CHW), system 1b (system 1 with purchased heating), system 1c (system 1 with purchased CHW and purchased HW).  
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the hvac system is modeled as either Sys-1, Sys-1a, Sys-1b, Sys-1c, or Not_Sys_1 in the B_RMR.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+- **terminal_unit_id_list**: list of terminal unit IDs associated with the HVAC system to be evaluated. These are sent to this function from the mater get_baseline_system_types function.
+- **zone_id_list**: list of zone IDs associated with the HVAC system to be evaluated. These are sent to this function from the mater get_baseline_system_types function.
+
+**Returns:**  
+- **is_baseline_system_1**: The function returns either Sys-1, Sys-1a, Sys-1b, Sys-1c, or Not_Sys_1 string output which indicates whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 1 (PTAC), system 1a (system 1 with purchased CHW), system 1b (system 1 with purchased heating), system 1c (system 1 with purchased CHW and purchased HW).  
+ 
+**Function Call:** 
+1. is_hvac_sys_heating_type_fluid_loop()
+2. is_hvac_sys_cooling_type_DX()
+3. is_hvac_sys_fan_sys_CV()  
+4. is_hvac_sys_fluid_loop_attached_to_boiler()
+5. is_hvac_sys_fluid_loop_purchased_heating()
+6. is_hvac_sys_fluid_loop_purchased_CHW()
+7. is_hvac_sys_cooling_type_fluid_loop()
+8. are_all_terminal_heat_sources_none_or_null()  
+9. are_all_terminal_cool_sources_none_or_null() 
+10. are_all_terminal_fans_null()  
+11. does_each_zone_have_only_one_terminal()    
+12. does_hvac_system_serve_single_zone()  
+13. is_baseline_system_1c()
+14. are_all_terminal_types_CAV() 
+ 
+## Logic:    
+- Create an object associated with the hvac system: `hvac_b = hvac_b.id`  
+- Set is_baseline_system_1 = Not_Sys_1: `is_baseline_system_1 = "Not_Sys_1"`    
+- Check that there is no preheat system, if there is none then carry on: `if len(hvac_b.preheat_system) == Null or hvac_b.preheat_system[0].heating_system_type == "NONE" :`    
+    - Check if the system is NOT system 1c, else (i.e., it is System 1c) is_baseline_system_1 = "SYS-1c": `if is_baseline_system_1c(B_RMR, hvac_b.id,terminal_unit_id_list,zone_id_list) == FALSE:`           
```

Comment:
Can we change the name of this function to is_baseline_system_1a_1b and not remove line 34? If it stays in, then I think we need to remove row 43 which results in mapping to SYS-1c.

Reply:
Discussed on call fine as is